services/configuration/CHANGELOG.md

```diff
@@ -52,6 +54,8 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - Implemented error handling based on error codes (#496).
 
+- Update to the [Exonum 0.6.0] release (#533).
```

Comment:
I don't really think that this is a new feature; I'd rather characterize it as "internal improvement".

Reply:
I think we should not mention that fact at all

Reply:
It is controversial, but it can often be the only change in a new release, so I prefer to leave it. Plus it is (currently) always a breaking change, so I'm not sure that "internal improvements" section is a good place for that.